cameronRomo/GameStack
# Enhacements/UI improvements

## Description

Adds UI improvements to enhance the user experience.

## Related Issue

closes #10 

## Motivation and Context

There was some awkward rendering, too many loading icons, and some missing headers that could be improved on.

## How Has This Been Tested?

The dynamic heading has been tested, and stubbing for fetching genres removed now that the genre list is shipped with the app.

To manually test:
1. Confirm that the background for dark mode is a darker grey
2. Confirm that Genres and platforms load automatically, and are no longer asynchronous
3. When selecting a genre the game's header should read "<genre> Games"
4. When selecting a platform the game's header should read "<platform> Games"
5. When selecting a genre and a platform "<platform> <genre> Games"
6. Massively Multiplayer should no longer overlap with the image, and wrap the text
7. Genre images should have better resolution (displayed as covers)

## Screenshots (if appropriate):
<img width="1420" alt="Screen Shot 2023-05-09 at 2 30 57 PM" src="https://github.com/cameronRomo/GameStack/assets/63012953/7cbc820d-7720-4452-9815-4a60c1283cce">

## Types of changes




## Checklist:




- [x] UI changes are responsive.
